migrations/20200915113210_van_activist_codes.js

```diff
@@ -80,7 +80,8 @@ exports.up = function(knex) {
           'api_key', json_build_object('__secret', v_api_key_ref),
           'van_system_id', queue_refresh_van_activist_codes.van_system_id,
           '__after', 'insert_van_activist_codes'
-        )
+        ),
+        'van-api'
```

Comment:
This could cause a refresh to be delayed while a 100k sync runs - let's have:
`van-api-refresh` and `van-api-sync`